src/esp/gfx/RenderTarget.h

```diff
@@ -38,11 +38,11 @@ class RenderTarget {
      */
     RgbaBuffer = 1 << 0,
     /**
-     * create objectId buffer
+     * create objectId texture
      * No need to set it for color sensor, depth sensor etc. as it makes the
      * rendering slower
      */
-    ObjectIdBuffer = 1 << 1,
+    ObjectIdTexture = 1 << 1,
     /**
      * @brief use depth texture, it must be set for the depth sensor.
      * No need to set it for color sensor, objectId sensor etc. as it makes the
```

Comment:
Same here, rename them all to `*Attachment` to not need to differentiate between renderbuffers and textures?